src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1343,6 +1343,12 @@ void TorrentHandle::handleStorageMovedFailedAlert(libtorrent::storage_moved_fail
 
 void TorrentHandle::handleTrackerReplyAlert(libtorrent::tracker_reply_alert *p)
 {
+    if (m_previousWarningUrl == p->url)
```

Comment:
Follow this guidelines => https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#b-other-code-blocks